### ISSUE TITLE
Prioritize Windows features during search

### ISSUE THREAD

**Issue by derei** [stnkl/EverythingToolbar#228]
When searching for things like calculator, command prompt, power shell, I get a lot of results and they are not the ones I want. So, I must be really specific when searching (eg. powershell.exe), which is time consuming. Also it shows a lot of results, as there are various places where windows stores a "powershell.exe" file.

What I would like is when searching for windows features (Settings, for example), or executables such as powershell, to receive those results with priority, and also to be able to assign aliases to them, for the purpose to shorten the typing.

Basically, to include some of the functionality of the native Windows search function. I understand that native search can still be accessed (this is how I am doing it now), but it would be even better to not have to rely on two different search functions.

**Comment by apzc2529** [comment]
Yes, it is very useful to list cmd, calculator prior to others.  Another method may be to show these two search windows at the same time.

**Comment by derei** [comment]
it's been more than one month since i posted this issue, but for some reason it doesn't seem to trigger interest. Is it only me using this plugin the wrong way, by trying to search for relevant things?
All others use it the right way by searching for nonsense?

**Comment by masiv1001** [comment]
> it's been more than one month since i posted this issue, but for some reason it doesn't seem to trigger interest. Is it only me using this plugin the wrong way, by trying to search for relevant things? All others use it the right way by searching for nonsense?

I was searching the same thing, I'm currently using this and Windows search, and it would be nice to at least being able to prioritize .lnk files (those are the programs in start menu mainly), I tried power toy's search but it's integration with everything lacks performance so I dropped using it for launching programs.

**Comment by stnkl** [comment]
Sorry for the late response. This has been requested before in #186. You can also find the current best solution of searching for only applications there.
